apicapi/apic_client.py

```diff
@@ -25,6 +26,9 @@
 import requests
 import requests.exceptions as rexc
 
+from OpenSSL.crypto import FILETYPE_PEM, load_privatekey, sign
+import base64
```

Comment:
Per PEP8, The import convention is as follows:
[block-1: standard libraries]
<withe line>
[block-2: 3rd party libraries]
<withe line>
[block-3: this project (apicapi.*...)]

In each block, the imports should be in alphabetical order, and you should always import a module (that you can then use to access the class/attribute/method that you need).